src/icons/folderIcons.ts

```diff
@@ -597,7 +597,7 @@ export const folderIcons: FolderTheme[] = [
       { name: 'folder-job', folderNames: ['job', 'jobs'] },
       {
         name: 'folder-keys',
-        folderNames: ['keys', 'key', 'token', 'tokens', 'jwt'],
+        folderNames: ['keys', 'key', 'token', 'tokens', 'jwt', 'secret', 'secrets'],
```

Comment:
Some formatting is missing here. As the array has to many items, there should be a line break after each item. I'd recommed to checkout the repository with VS Code locally or with Codespaces and install the Prettier Plugin. With it you can format and save the file in a correct way.